### PR TITLE
fix blitz new for people without global git config (patch)

### PR DESCRIPTION
**Pull request by akbo**
Closes: #1843

### What are the changes and their implications?

Set custom user.name and user.email for initial git commit in `blitz new`, so people who don't have a global git config don't experience an error.

### Checklist

- [ ] Changes covered by tests (tests added if needed) -> tested manually as discussed in #1843 
- [ ] PR submitted to [blitzjs.com](https://github.com/blitz-js/blitzjs.com) for any user facing changes -> no user facing changes

### CODE REVIEW

**Comment by akbo** [comment]
Looks like the CI tests fail at a place that is unrelated to my pull request?

**Comment by flybayer** [comment]
@akbo thanks! Yeah the test that failed is because of your change. Those generator tests need updated to reflect the new arguments.

**Comment by akbo** [comment]
Ok, I think I got it now. Sorry for the confusion!

File: packages/generator/src/generators/app-generator.ts [hunk 1]
```diff
@@ -238,7 +238,17 @@ export class AppGenerator extends Generator<AppGeneratorOptions> {
...
+        [
+          "-c",
+          "user.name='Blitz.js CLI'",
+          "-c",
+          "user.email='noop@blitzjs.com'",
+          "commit",
+          "--no-gpg-sign",
+          "--no-verify",
+          "-m",
+          "New baby Blitz app!",
```
Comment (flybayer) [inline packages/generator/src/generators/app-generator.ts:250]:
```suggestion
          "Brand new Blitz app!",
```

File: packages/generator/test/generators/app-generator.test.ts [hunk 2]
```diff
@@ -119,7 +119,17 @@ describe("AppGenerator", () => {
...
+      [
+        "-c",
+        "user.name='Blitz.js CLI'",
+        "-c",
+        "user.email='noop@blitzjs.com'",
+        "commit",
+        "--no-gpg-sign",
+        "--no-verify",
+        "-m",
+        "New baby Blitz app!",
```
Comment (flybayer) [inline packages/generator/test/generators/app-generator.test.ts:131]:
```suggestion
        "Brand new Blitz app!",
```

**Review by flybayer — APPROVED** [review]

**Comment by flybayer** [comment]
Sweet, thank you!!

@all-contributors add @akbo for code

**Pull request merged**